extensions/data-transfer/portability-data-transfer-backblaze/src/test/java/org/datatransferproject/datatransfer/backblaze/common/BackblazeDataTransferClientTest.java

```diff
@@ -53,192 +55,180 @@
-        // Arrange
-        createValidBucketList();
-        when(s3Client.putObject(any(PutObjectRequest.class), any(RequestBody.class)))
-                .thenThrow(AwsServiceException.builder().build());
-        BackblazeDataTransferClient client = createDefaultClient();
-        client.init(KEY_ID, APP_KEY, EXPORT_SERVICE);
-        // Act
-        client.uploadFile(FILE_KEY, testFile);
-        // Assert: expected exception
-    }
-
-    @Test
-    public void testUploadFileMultipart() throws BackblazeCredentialsException, IOException {
-        // Arrange
-        final String expectedVersionId = "123";
-        createValidBucketList();
-        when(s3Client.createMultipartUpload(any(CreateMultipartUploadRequest.class)))
-                .thenReturn(CreateMultipartUploadResponse.builder().uploadId("xyz").build());
-        when(s3Client.uploadPart(any(UploadPartRequest.class), any(RequestBody.class)))
-                .thenReturn(UploadPartResponse.builder().build());
-        when(s3Client.completeMultipartUpload(any(CompleteMultipartUploadRequest.class)))
-                .thenReturn(CompleteMultipartUploadResponse.builder().versionId(expectedVersionId).build());
-        final long partSize = 10;
-        final long fileSize = testFile.length();
-        final long expectedParts = fileSize / partSize + (fileSize % partSize == 0 ? 0 : 1);
-        BackblazeDataTransferClient client =
-                new BackblazeDataTransferClient(monitor, backblazeS3ClientFactory, fileSize / 2, partSize);
+
+  @Mock
+  private Monitor monitor;
+  @Mock
+  private BackblazeS3ClientFactory backblazeS3ClientFactory;
+  @Mock
+  private S3Client s3Client;
+  private static File testFile;
+  private static final String KEY_ID = "keyId";
+  private static final String APP_KEY = "appKey";
+  private static final String EXPORT_SERVICE = "exp-serv";
+  private static final String FILE_KEY = "fileKey";
+  private static final String VALID_BUCKET_NAME = EXPORT_SERVICE + "-data-transfer-bucket";
+
+  @BeforeClass
+  public static void setUpClass() {
+    testFile = new File("src/test/resources/test.txt");
+  }
+
+  @Before
+  public void setUp() {
+    when(backblazeS3ClientFactory.createS3Client(anyString(), anyString(), anyString()))
+        .thenReturn(s3Client);
+  }
+
+  private void createValidBucketList() {
+    Bucket bucket = Bucket.builder().name(VALID_BUCKET_NAME).build();
+    when(s3Client.listBuckets()).thenReturn(ListBucketsResponse.builder().buckets(bucket).build());
+  }
+
+  private void createEmptyBucketList() {
+    when(s3Client.listBuckets()).thenReturn(ListBucketsResponse.builder().build());
+  }
+
+  private BackblazeDataTransferClient createDefaultClient() {
+    return new BackblazeDataTransferClient(monitor, backblazeS3ClientFactory, 1000, 500);
+  }
+
+  @Test
+  public void testWrongPartSize() {
+    Assertions.assertThrows(IllegalArgumentException.class, () -> {
```

Comment:
Could you please use a static import for `assertThrows`? We use static imports for other asserts, so would be nice to keep the code consistent. It applies to every file where we use static imports for assertion functions.